loopflow-computation/src/main/java/com/farao_community/farao/loopflow_computation/LoopFlowComputation.java

```diff
@@ -199,10 +199,10 @@ private static void buildFlowFromNetwork(Network network, Crac crac, Map<Cnec, D
 
     public Map<String, Double> buildLoopFlowsFromReferenceFlowAndLoopflowShifts(Map<Cnec, Double> frefResults, Map<Cnec, Double> loopFlowShifts) {
         Map<String, Double> loopFlows = new HashMap<>();
-        for (Map.Entry<Cnec, Double> entry : frefResults.entrySet()) {
+        for (Map.Entry<Cnec, Double> entry : loopFlowShifts.entrySet()) {
             Cnec cnec = entry.getKey();
-            double loopflowShift = requireNonNull(loopFlowShifts.get(cnec), () -> "No loopflowShifts value for " + cnec.getId());
-            loopFlows.put(cnec.getId(), entry.getValue() - loopflowShift);
+            double fref = requireNonNull(frefResults.get(cnec), () -> "No reference flow value for " + cnec.getId());
+            loopFlows.put(cnec.getId(), fref - entry.getValue());
```

Comment:
why did you change this?

Reply:
there is non null  check in one but not in the other map